### PR TITLE
chore: add go version 1.12 to travis

### PR DESCRIPTION
**Pull request by firfircelik**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,4 +13,4 @@ script:
   - ./go.test.sh
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+- bash <(curl -s https://codecov.io/bash)
```
Comment (ferhatelmas) [inline .travis.yml]: why change here ?

Comment (firfircelik) [reply .travis.yml]: not intentional

Comment (ferhatelmas) [reply .travis.yml]: please be more careful, seems you fixed but there is still unnecessary whitespace change

Comment (firfircelik) [reply .travis.yml]: I am but it seems when I save it in vscode something goes wrong. But will be much more carefull. Thanks for advice.

**Review by ferhatelmas — APPROVED** [review]

**Pull request merged**